cmd/kubectl-gadget/main.go

```diff
@@ -125,7 +128,22 @@ func main() {
 	}
 
 	if !skipInfo {
+		kubectlGadgetVersion := version.Version()
 		grpcRuntime.InitDeployInfo()
+		info, err := deployinfo.Load()
+		if err != nil {
+			log.Warnf("failed to load deploy info: %s", err)
+		}
+		serverSemver, err := semver.Make(info.ServerVersion)
+		if err != nil {
+			log.Errorf("Failed to create a semantic version object for the gadget pods version: %s", err)
+		}
+		if !kubectlGadgetVersion.Equals(serverSemver) {
```

Comment:
There should also be a check on `len(kubectlGadgetVersion.Pre) == 0` to avoid printing this version when it's not a release. (See logic on cmd/kubectl-gadget/utils/trace.go)

Reply:
@mauriciovasquezbernal can gadgetctl have a pre release?

Reply:
Yes, the versioning logic is the same for all components.